python/CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for tchannel.py
 =========================
 
+0.9.2 (unreleased)
```

Comment:
@abg @junchaowu @abg just to make sure we're all on the same page:

X.Y.Z - major.minor.patch

Every piece of new functionality should result in a new minor version increment. If the major is still 0, aka we're still in beta, then that functionality can contain breaking changes. This means that applications should lock the version to the minor, and also means we won't be able to ship new functionality to them. 

Once we are 1.0, new functionality still gets shipped in the minor, aka 1.1, 1.2, 1.3, etc... but breaking changes will force us to cut a 2.0.

Patching existing functionality, aka fixing bugs, should always increment the patch version. 

This diff represents new functionality, so the correct next version is `0.10.0`. This is super important because we have customers locked on `tchannel>=0.9,<0.8` that are only expecting bug fixes, no new & possibly breaking changes. 

Makes sense?